src/CabanaPD_Types.hpp

```diff
@@ -40,5 +40,9 @@ struct LinearLPS
 {
 };
 
+struct DynamicCriticalStretch
```

Comment:
We may move the DynamicCriticalStretch to a different PR.

Reply:
Opened in #117